src/mono/msbuild/android/build/AndroidApp.InTree.props

```diff
@@ -0,0 +1,13 @@
+<Project>
```

Comment:
How does this file get consumed/used? There aren't references to it right?

Reply:
This is around for projects that want to decouple themselves from the testing targets.  The Android sample might be a good candidate.